package.json

```diff
@@ -6,7 +6,16 @@
   "scripts": {
     "build": "yarn ts-node node_modules/.bin/webpack -c ./webpack.config.ts",
     "dev": "yarn ts-node node_modules/.bin/webpack serve -c ./webpack.config.ts --progress",
-    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}' -I '/node_modules/(?!(@fusion)/)/'"
+    "ts-node": "ts-node -O '{\"module\":\"commonjs\"}' -I '/node_modules/(?!(@fusion)/)/'",
+    "lint": "eslint packages/*/src --config .eslintrc.js --ext .js,.ts,.tsx  --color --cache",
+    "prepare": "husky install"
+  },
+  "lint-staged": {
+    "packages/**/*.{js,ts,tsx,mts}": [
+      "yarn lint"
+    ],
+    "packages/**/*.scss": "stylelint",
+    "**/*": "prettier --write --ignore-unknown"
```

Comment:
can we add it in "scripts" section ?? basically a command for this, just like "yarn lint".

Reply:
sure